envs/monkey_zoo/blackbox/config_templates/powershell.py

```diff
@@ -0,0 +1,21 @@
+from copy import copy
+
+from envs.monkey_zoo.blackbox.config_templates.base_template import BaseTemplate
+from envs.monkey_zoo.blackbox.config_templates.config_template import ConfigTemplate
+
+
+class PowerShell(ConfigTemplate):
+    config_values = copy(BaseTemplate.config_values)
+
+    config_values.update(
+        {
+            "basic.exploiters.exploiter_classes": ["PowerShellExploiter"],
+            "basic_network.scope.subnet_scan_list": ["10.2.2.45", "10.2.3.47"],
+            "basic.credentials.exploit_password_list": ["Passw0rd!", ""],
```

Comment:
We shouldn't have a blank password in the password list. It's up to the exploiter to know to try a blank password.